backend/src/main/java/capstone/backend/controller/OrderToSupplierController.java

```diff
@@ -0,0 +1,29 @@
+package capstone.backend.controller;
+
+
+import capstone.backend.model.dto.order.OrderToSupplierDTO;
+import capstone.backend.services.OrderToSupplierService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/orders_suppliers")
+public class OrderToSupplierController {
+
+    private final OrderToSupplierService service;
+
+    public OrderToSupplierController(OrderToSupplierService service) {
```

Comment:
If you annotate the class with lombok's `@RequiredArgsConstructor` you can delete the constructor.

Reply:
if I remember correctly we were taught by the coaches to only use Lombok's annotators for our Models

Reply:
I get why certain annotation make more sense for certain kind of classes. But I don't see why one should limit lombok in general only to models.
What reason was given to you?